src/components/calculator/FirstTimeUserIntroduction.tsx

```diff
@@ -27,19 +27,18 @@ export function FirstTimeUserIntroduction(): React.ReactElement {
         </p>
       </Expandable>
       <Alert variant="secondary">
-        <strong>
-          <BsCheckBox /> We would love your feedback:
-        </strong>{' '}
-        We want your help to make the microCOVID calculator as helpful as
-        possible!{' '}
-        <a
-          href="https://forms.gle/WzFWcmyXwQMNRqGa7"
-          target="_blank"
-          rel="noreferrer"
-        >
-          Please fill out our user survey
-        </a>{' '}
-        to help us prioritize new features.
+        <BsCheckBox />
```

Comment:
A bit of a trickery is needed to keep the space between the checkbox and the text:
```suggestion
        <BsCheckBox />{' '}
```